_docs/delegators.md

```diff
@@ -1,23 +1,20 @@
 ---
-title: Delegate your stake to a collotor
-subtitle: Stake as a delegator
+title: Delegator Overview
+subtitle: How to stake to a collator and earn rewards
 author: charles
-tags: [delegate, delegator, stake, collotor]
+tags: [delegator]
 ---
 
-# Introduction
+## Risks
```

Comment:
@chrisli30 - do we need legal to approve this?

Reply:
We can ask them to take a look once we publish this page. I think it’s not required since there’s no "sale" stuff in it.